packages/sdk/chainDeploy/mainnets/bsc.ts

```diff
@@ -290,6 +290,19 @@ export const deploy = async ({ run, ethers, getNamedAccounts, deployments }: Cha
   const mpo = await ethers.getContract("MasterPriceOracle", deployer);
   const nativeBnb = underlying(assets, assetSymbols.BNB);
 
+  // Wombat Lp Token Price Oracle
+
+  const wombatOracle = await deployments.deploy("WombatLpTokenPriceOracle", {
+    from: deployer,
+    args: [mpo.address],
```

Comment:
please remove the `mpo.address` param and the ` waitConfirmations: 1,` config line